### PR TITLE
Feature/email without tracking habit

### PR DESCRIPTION
**Pull request by jvargas98**
## Explain why this change is being made.
As a registered user with an active habit, I want to receive an email when I have more than four days without tracking a habit so that I can add my progress to the habit.

---

## Explain how this is accomplished.
the register user with an active habit who has not tracked his habit for more than 4 days, should receive an email reminding him to track his habit.

---

## How do you manually test this?
1. As a registered user with a habit saved.
2. Dont track this habit for more than 4 days,
3. You receive an email.

---

### CODE REVIEW

**Review by LoginALee — APPROVED** [review]
Good one!

**Review by GregVaz — APPROVED** [review]
:rocket:

**Review by ese-varo — APPROVED** [review]
:rocket:

File: app/jobs/reminder_habit_tracking_job.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        days = habit.days.where('date < ?', Date.today).last(4)
+        HabitTrackingMailer.with(habit: habit).reminder.deliver_later if evaluate_days(days)
+      end
+    end
+  end
+
+  def evaluate_days(days)
+    sum = 0
+    days.map { |day| sum += 1 unless day.status }
+    sum == 4
```
Comment (OmauryMarmolejo) [inline app/jobs/reminder_habit_tracking_job.rb]: You can split this into two methods

**Review by OmauryMarmolejo — APPROVED** [review]

File: app/jobs/reminder_habit_tracking_job.rb (hunk 1 above)
Comment (jvargas98) [reply app/jobs/reminder_habit_tracking_job.rb]: Thanks for the comment

**Pull request merged**